docusaurus/docs/Android/04-compose/05-message-components/04-message-composer.mdx

```diff
@@ -171,7 +173,7 @@ fun MyCustomComposer() {
                         verticalAlignment = Alignment.CenterVertically
                     ) {
                         Icon(
-                            imageVector = Icons.Default.Keyboard,
```

Comment:
The keyboard icon seems to have been removed.
Not sure when but probably pre-stable compose release.

https://developer.android.com/jetpack/compose/graphics/images/material